bb/master/config.py

```diff
@@ -19,10 +19,14 @@
 # SOFTWARE.
 
 import sys
+import os
 from enum import Enum
 
 import msdk_secrets
 
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Comment:
Please, check. I guess that it will append incorrect path. 
Need to get infrastructure directory.